test/gpu_tsit5_tests.jl

```diff
@@ -0,0 +1,121 @@
+using StaticArrays, SimpleDiffEq, BenchmarkTools, DiffEqGPU, CUDA
```

Comment:
This can be removed and the MWE usage can be done in the current test files.

Reply:
This wasn't done.

Reply:
It's already done https://github.com/SciML/DiffEqGPU.jl/pull/148/files#diff-6649bc3d1b6d365552d9121ef6e55dada7f1159fb0be5fd0095470074ace4b3c